README.md

```diff
@@ -708,7 +721,7 @@ tugboat:php:init:
     - docker-php-ext-install ldap
 ```
 
-You can additionally add an `online` step by adding a task named `online:tugboat`
+You can additionally add an `online` step by adding a task named `tugboat:online`
```

Comment:
"You can additionally add" sounds redundant.
But I'm not an English teacher :P

Reply:
How about "You can additionally include ..."